### ISSUE TITLE
[BottomNavigation] BottomNavigationTypicalUseExample tabs missing accessibilityHint

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#3180]
### Overview
The tabs in the BottomNavigationTypicalUseExample view controller aren't providing hints.

### Reproduction steps
1. Turn on VoiceOver
1. Navigate to each tab

**Expected Result**
Each tab has an accessibility hint value

**Actual Result**
Tabs have labels but not hints.

### Version number
50.0.0

**Comment by andrewoverton** [comment]
@romoore , do we still feel like this is an issue? Right now VoiceOver reads "Favorites, Tab, Tab, 3 of 5", for example. Do we need a hint in there?

**Comment by romoore** [comment]
Oh noes, it reads "tab, tab"? Let's close this issue but open one to fix what voiceover reads.
